dockerfiles/init/modules/k8s/files/scripts/k8s.yml

```diff
@@ -0,0 +1,265 @@
+---
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    labels:
+      app: che
+    name: che
+  data:
+    CHE_HOST: "192.168.99.100.nip.io"
```

Comment:
do you have any plans to get rid of these hardcodes?

Reply:
Yes, it is minikube IP on my machine. 
1. For the time we don't have other installations to test this stuff
1. I didn't want to use shell script to deploy Che on k8s since it doesn't seem right to me, so we probably need to wait for helm deployment instructions which would allow us to customize this, I hope

Reply:
But if you have a suggestion how to customize this properly with acceptable defaults and without shell script your help would be very appreciated.